cScripts/functions/modules/fn_zenModule_rekitVehicle.sqf

```diff
@@ -33,16 +33,22 @@
             _objectPos setVariable [QEGVAR(Vehicle,Type), nil, true];
         };
 
+        _objectPos setVariable [QEGVAR(Vehicle,Functions), nil, true];
         _objectPos setVariable [QEGVAR(Vehicle,Inventory), nil, true];
+        _objectPos setVariable [QEGVAR(Vehicle,PylonLoadout), nil, true];
+        _objectPos setVariable [QEGVAR(Vehicle,Cosmetics), nil, true];
+        _objectPos setVariable [QEGVAR(Vehicle,StagingActions), nil, true];
+        _objectPos setVariable [QEGVAR(Vehicle,Radio), nil, true];
 
-        
         private _cargoArray = _objectPos getVariable ["ace_cargo_loaded",[]];
 
         {   
             [_x, _objectPos] call ace_cargo_fnc_removeCargoItem;
             
         } forEach _cargoArray;
 
+        sleep 0.05;
```

Comment:
use any of this two instead:
```cpp
[{
    _this call EFUNC(vehicle,addFunctions);
    _this call EFUNC(vehicle,addInventory);
    _this call EFUNC(vehicle,addPylonLoadout);
    _this call EFUNC(vehicle,addCosmetics);
    _this call EFUNC(vehicle,addStagingActions);
    _this call EFUNC(vehicle,addRadio);
}, [_objectPos], 0.05] call CBA_fnc_waitAndExecute;
```
https://cbateam.github.io/CBA_A3/docs/files/common/fnc_waitAndExecute-sqf.html


```cpp
[{
    _this call EFUNC(vehicle,addFunctions);
    _this call EFUNC(vehicle,addInventory);
    _this call EFUNC(vehicle,addPylonLoadout);
    _this call EFUNC(vehicle,addCosmetics);
    _this call EFUNC(vehicle,addStagingActions);
    _this call EFUNC(vehicle,addRadio);
}, [_objectPos]] call CBA_fnc_execNextFrame;
```
https://cbateam.github.io/CBA_A3/docs/files/common/fnc_execNextFrame-sqf.html